server/views/pages/reportDetailMacro.njk

```diff
@@ -6,18 +6,27 @@
   <span data-qa="{{ dataQa }}"> {{ value }}</span>  
 {% endmacro %}
 
-{% macro reportHeading(data) %}
-  <div class="govuk-!-margin-bottom-1">
-    {{ fieldValue('Created by', data.reporterName, 'reporter-name')  }}
-    <br/>
-    {{ fieldValue('Date and time', data.submittedDate | formatDate('D MMMM YYYY, HH:mm'), 'submitted-date')  }}
-    <br/>
-    {{ fieldValue('Prisoner name', data.offenderDetail.displayName, 'prisoner-name')  }}
-    <br/>
-    {{ fieldValue('Prisoner number', data.offenderDetail.offenderNo, 'prisoner-number')  }}
-    <br/>
-    {{ fieldValue('Incident number', data.incidentId, 'incident-number')  }}
-  </div>    
+{% macro reportHeading(data, print) %}
+  <div class="govuk-grid-row  {% if print %} flex-container {% endif %}">
+      <div class="govuk-grid-column-one-third govuk-!-margin-bottom-4" style="flex-basis: 25%">
+        <p class="govuk-!-margin-top-0 govuk-!-margin-bottom-0 govuk-label--s">Created by</p>
+        <p class="govuk-!-margin-top-0 govuk-!-margin-bottom-0 govuk-label--s">Date and time</p>
+        <p class="govuk-!-margin-top-0 govuk-!-margin-bottom-0 govuk-label--s">Prisoner name</p>
+        <p class="govuk-!-margin-top-0 govuk-!-margin-bottom-0 govuk-label--s">Prisoner number</p>
+        {% if not print  %}
+          <p class="govuk-!-margin-top-0 govuk-!-margin-bottom-0 govuk-label--s">Incident number</p>
+        {% endif %}
+      </div>
+      <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-4" style="flex-basis: 75%" >
+        <p class="govuk-!-margin-top-0 govuk-!-margin-bottom-0" data-qa="reporter-name"> {{ data.reporterName }} </p>
```

Comment:
You created that custom class: `margin-top-bottom-0`, is there a reason you aren't using it here but are in other places?

Reply:
after our chat about css, I thought it may be better to use just the govuk classes, even though it would mean more code in the html. I am thinking that perhaps weed out any custom classes in due course from the whole app?